src/include/network/lan/udev.rb

```diff
@@ -82,13 +82,11 @@ def RemoveKeyFromUdevRule(rule, key)
     # Tripplet is given as a string in form KEY="VALUE" or
     # MATCHKEY=="MATCHVALUE"
     def AddToUdevRule(rule, tripplet)
-      rule = deep_copy(rule)
-      if !Builtins.regexpmatch(tripplet, ".+={1,2}\".*\"")
-        return deep_copy(rule)
-      end
+      return rule unless tripplet =~ /.+={1,2}\".*\"/
+
       rule = [] if rule.nil?
 
-      Builtins.add(rule, tripplet)
+      rule << tripplet
```

Comment:
thanks for pointing that.I was unsure of this change. I've already checked all occurrences in the network code and it should work fine. However as I think of it now, I'll return to old behavior as it is probably more useful.